libsolidity/interface/OptimiserSettings.h

```diff
@@ -55,7 +55,7 @@ struct OptimiserSettings
 			"xa[rul]"                  // Prune a bit more in SSA
 			"xa[r]cL"                  // Turn into SSA again and simplify
 			"gvif"                     // Run full inliner
-			"CTUca[r]LsTFOtfDnca[r]Iulc" // SSA plus simplify
+			"CTUca[r]LsSTFOtfDnca[r]Iulc" // SSA plus simplify
```

Comment:
We should check if we have inlined literals right before executing the eliminator.